hibernate-cache/src/test/java/org/infinispan/test/hibernate/cache/functional/cluster/NaturalIdInvalidationTest.java

```diff
@@ -55,9 +59,8 @@
 	}
 
 	@Test
-   @Ignore("Randomly failing on CI - ISPN-8114")
 	public void testAll() throws Exception {
-		log.info( "*** testAll()" );
+      log.infof("*** %s", name.getMethodName());
```

Comment:
TestNG only prints `Test starting: testAll()`, leaving out all the test parameters, so trying to figure out which testAll is being executed becomes quite a mission. I've personally had to go through this pain analysing TRACE logs.

In other tests, the thread name has been adjusted to have the parameters embedded within the test name. However, in this particular test this does not seem to be easy to do (all threads are named `Thread-xxx`... something to do with how the test is designed... Changing this seemed messy, so instead went for the simpler fix, which is to print out not only the test method name but also the test parameters. This is enough to figure which `Thread-` belongs to what.

If you know how to fix it, be my guest, creating a separate JIRA for it :)